advocacy_docs/tools/alteruser_utility/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: "Alteruser utility"
+description: "How to use the alteruser utility to alter a PostgreSQL role in your EDB Postgres Advanced Server databases"
+iconName: Tools
+deepToC: true
+---
+
+## Overview
+
+The `alteruser` utility modifies an existing PostgreSQL role in your EDB Postgres Advanced Server's clusters. It's a wrapper around the SQL command ALTER ROLE and is similar to the `createuser` utility. 
```

Comment:
The original text said "in line with" the createuser utility. Is this what was meant?

Reply:
yes